rotkehlchen/serialization/schemas.py

```diff
@@ -133,7 +184,11 @@ def transform_data(self, data: dict[str, Any], **_kwargs: Any) -> dict[str, Cryp
         return {'crypto_asset': crypto_asset}
 
 
-class EvmTokenSchema(BaseCryptoAssetSchema, RequiredEvmAddressSchema):
+class EvmTokenSchema(
+    AssetWithOraclesSchema,
+    CryptoAssetFieldsSchema,
```

Comment:
This thing with the schemas looks confusing and seems that need to be tough a bit better. For example can't `CryptoAssetFieldsSchema` inherit from `AssetWithOraclesSchema` and then you can avoid having `AssetWithOraclesSchema` both in crypto and evm schemas?

Reply:
Yes, I agree that it's a bit hard to follow the schemas right now. Rearranged it a bit. Hope it became more readable